src/app/settings/page.tsx

```diff
@@ -0,0 +1,389 @@
+"use client";
+
+import { useState, useEffect } from "react";
+import Sidebar from "../../components/sidebar";
+import SettingBar from "@/components/SettingBar";
+import { toast } from "react-hot-toast";
+import { FileText } from "lucide-react";
+import { SaveAll } from "lucide-react";
+import { User } from "lucide-react";
+import { Bell } from "lucide-react";
+import { Shield } from "lucide-react";
+import { Settings } from "lucide-react";
```

Comment:
You can use 1 import and then add up all in parenthesis, no need to import again and again.